partner-management-default.properties

```diff
@@ -310,21 +310,23 @@ mosip.role.pms.getmisplicensekey=PARTNERMANAGER,PARTNER_ADMIN
 mosip.role.pms.postmispfiltervalues=MISP_PARTNER,MISP,PARTNERMANAGER,PARTNER_ADMIN
 mosip.role.pms.postmispsearch=MISP_PARTNER,MISP,PARTNERMANAGER,PARTNER_ADMIN
 
-# query param usd to refer url to redirect after logout 
-mosip.iam.post-logout-uri-param-key=post_logout_redirect_uri 
-# end session endpoint in OIDC 
-mosip.iam.end-session-endpoint-path=/protocol/openid-connect/logout 
-# Esignet claims mapping file url 
-mosip.pms.esignet.claims-mapping-file-url=${spring_config_url_env}/*/${active_profile_env}/${spring_config_label_env}/identity-mapping.json
-# Esignet amr-acr mapping file url 
-mosip.pms.esignet.amr-acr-mapping-file-url=${spring_config_url_env}/*/${active_profile_env}/${spring_config_label_env}/amr-acr-mapping.json
-## Esignet Service apis for create and update 
+
+# query param usd to refer url to redirect after logout
+mosip.iam.post-logout-uri-param-key=post_logout_redirect_uri
+# end session endpoint in OIDC
+mosip.iam.end-session-endpoint-path=/protocol/openid-connect/logout
+
+# esignet claims mapping file url
+mosip.pms.esignet.claims-mapping-file-url=https://raw.githubusercontent.com/mosip/mosip-config/dev-integration/identity-mapping.json
+# esignet amr-acr mapping file url
+mosip.pms.esignet.amr-acr-mapping-file-url=https://raw.githubusercontent.com/mosip/mosip-config/dev-integration/amr-acr-mapping.json
```

Comment:
Are these change intentional ? What happens when a country change these file in their github repo ? The default assumption is that change will be default taken.

Reply:
@dhanendra06 to check this as well

Reply:
updated